tests/playwright/pages/wp-admin-page.js

```diff
@@ -105,4 +105,28 @@ module.exports = class WpAdminPage extends BasePage {
 		await this.page.selectOption( '#WPLANG', language );
 		await this.page.locator( '#submit' ).click();
 	}
+
+	async gotoThemesPage() {
+        await this.page.goto( '/wp-admin/themes.php', { waitUntil: 'networkidle' } );
```

Comment:
Suggestion: can you create a function called `activateHelloElementorTheme()` or `activateTheme( 'Hello Elementor' )`